code/datums/ruins/lavaland.dm

```diff
@@ -62,6 +62,8 @@
 	suffix = "lavaland_surface_syndicate_base1.dmm"
 	cost = 20
 	allow_duplicates = FALSE
+	// Debug
+	always_place = TRUE
```

Comment:
🥴

Reply:
This'll be removed when I take it out of draft 😅